src/test/java/unicash/testutil/TypicalTransactions.java

```diff
@@ -76,11 +76,11 @@ public class TypicalTransactions {
      * Returns a {@code UniCash} with all the typical transactions.
      */
     public static UniCash getTypicalUniCash() {
-        UniCash ab = new UniCash();
```

Comment:
Nice work catching this minor detail! Can we further change it to just uniCash or emptyUniCash to better align with the variable naming conventions? I doubt it would be an issue though, just trying to be extra careful with the coding convention requirements

Reply:
Good suggestion! I've modified it to `typicalUnicash` instead.